app/renderer/suggest.js

```diff
@@ -934,6 +934,151 @@ const suggestCommand = searchStr => {
             }
         }
     }
+    if ("bmadd".startsWith(command) && !confirm) {
+        const {
+            getBookmarkData, validBookmarkOptions
+        } = require("./bookmarks")
+        const bmData = getBookmarkData()
+        const optionsStr = args.join(" ")
+        const options = optionsStr.split("~")
+        const lastOption = options[options.length - 1]
+        if (lastOption.includes("=")) {
+            const otherOptions = options.slice(0, -1).filter(o => o)
+            let baseCmd = "bmadd"
+            if (otherOptions.length > 0) {
+                baseCmd += ` ${otherOptions.join("~")}`
+            }
+            const [key, value = ""] = lastOption.split("=")
```

Comment:
These names did not make sense at first when reading the code below, can we rename it to "optionName" and "optionValue" or similar?